lib/parser/provenance.cc

```diff
@@ -328,6 +360,43 @@ std::optional<ProvenanceRange> CookedSource::GetProvenanceRange(
   return {ProvenanceRange{first.start(), last.start() - first.start()}};
 }
 
+std::optional<CharBlock> CookedSource::GetCharBlockFromLineAndColumns(
+    int line, int column1, int column2) const {
+  auto provenanceStart = allSources_.GetFirstFileProvenance().value().start();
```

Comment:
We prefer the use of braced initializers; they're safer from dangerous conversions when the type is not `auto`, and consistent with other initializers when the type is `auto`.